src/main/asciidoc/index.adoc

```diff
@@ -1,3 +1,125 @@
 = Cassandra Client for Vert.x
 
 A Vert.x client allowing applications to interact with a Cassandra service
+
+
+== Getting Started
+
+For now, in order to use the client you need to build it locally first:
+
+----
+git clone https://github.com/Sammers21/vertx-cassandra-client
+cd vertx-cassandra-client
+mvn clean install -Dmaven.test.skip=true -s .travis.maven.settings.xml
+----
+
+=== Maven
+
+Add the following dependency to your maven project:
+
+[source,xml,subs="+attributes"]
+----
+<dependency>
+  <groupId>io.vertx</groupId>
+  <artifactId>vertx-cassandra-client</artifactId>
+  <version>${maven.version}</version>
+</dependency>
+----
+
+=== Gradle
+
+Add the following dependency to your gradle project:
+
+[source,groovy,subs="+attributes"]
+----
+dependencies {
+  compile 'io.vertx:vertx-cassandra-client:${maven.version}'
+}
+----
+
+=== Introduction
+
+http://cassandra.apache.org/[Cassandra] is a distributed system, and it can have many nodes.
+To connect to Cassandra you need to specify the addresses of some cluster nodes when creating `CassandraClientOptions` object:
+
+[source,$lang]
+----
+{@link examples.CassandraClientExamples#specifyingNodes}
+----
+
```

Comment:
Can you add a note about the default behavior? It will be useful for people trying this out with a local cluster.

Reply:
Done